src/vulkan/device.cc

```diff
@@ -767,6 +765,22 @@ Result Device::Initialize(
         return amber::Result("Missing compute full subgroups feature");
       }
     }
+
+    // If Vulkan 1.4 structure exists the features are set there.
```

Comment:
What happens if vulkan 1.4 isn't provided, does this mean it can no longer find the feature it previously found? Does the old way not work anymore, like is `VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_INDEX_TYPE_UINT8_FEATURES_EXT` no longer returned above so the pointer isn't set?